docs/elements.md

```diff
@@ -2317,10 +2317,12 @@ Displays the contents of question directions.
 
 #### Details
 
-Contents are only shown during question input portion. When a student
-either makes a submission or receives the correct answer, the information
-between these tags is hidden. If content exists outside of a question panel,
-then it will be displayed alongside the answer.
+Information between this tag represents the question text and is 
+always shown when the question is displayed to students. When a 
+student either makes a submission or receives the correct answer,
+the information between these tags is not shown in the 
+`submission-panel` and `answer-panel`. If content exists outside of a 
+question panel, then it will be displayed in the `answer-panel`.
```

Comment:
also I just realized this isn't true, it should be:

> If content exists outside of a question panel, then it will be displayed in the `answer-panel` **and the `submission-panel`**.